test-data/run.test

```diff
@@ -287,6 +287,36 @@ print(f((1,2,3,4)))
 [out]
 2
 
+[case testFunctionCallWithDefaultArgs]
+from typing import Tuple, List
+def f(x: int, y: int = 3, s: str = "test", l: List[int] = []) -> Tuple[int, str]:
```

Comment:
I wouldn't bother testing doing `[]`, since we do the wrong thing for it anyways.